Dockerfile

```diff
@@ -88,6 +88,7 @@ go install -v ./... && \
 mv $GOPATH/bin/sqlflowserver /usr/local/bin && \
 mv $GOPATH/bin/repl /usr/local/bin && \
 (cd python/couler && python setup.py -q install) && \
+(git clone https://github.com/wangkuiyi/couler.git && cd couler && python setup.py sdist bdist_wheel && pip install dist/*.whl) && \
```

Comment:
The repo name has been changed from `couler` to `fluid`.

Hi @Yancey1989 @wangkuiyi, will we be actively developing `fluid`? If so, should we move the `fluid` source code into this repo?

Reply:
Yes, we have some todo tasks on `fluid`, maybe using a separated repo can be helpful with the developing of `fluid`, and we can use a fixed commit ID in sqlflow, meanwhile optimizing the CI on `fluid` repo to make it more stable.